.adr-dir

```diff
@@ -1 +0,0 @@
-docs/architecture/decisions
```

Comment:
I was thinking about this in the past but I never got to the point to update the documentation. I believe there is some value in adding a few entries here with each milestone that we are working on. For example, the indexing rationale in the new ES architecture could be a great fit. The future work about the API would also be a good candidate. I guess what I am saying is that there is some value in keeping this section and start using it. Thoughts.

Reply:
Sure, if we think we're going to use it, let's leave it in